src/deps/uws.zig

```diff
@@ -784,95 +784,80 @@ extern fn us_socket_is_closed(ssl: i32, s: ?*Socket) i32;
 extern fn us_socket_close(ssl: i32, s: ?*Socket, code: i32, reason: ?*anyopaque) ?*Socket;
 extern fn us_socket_local_port(ssl: i32, s: ?*Socket) i32;
 extern fn us_socket_remote_address(ssl: i32, s: ?*Socket, buf: [*c]u8, length: [*c]i32) void;
+
 pub const uws_app_s = opaque {};
 pub const uws_req_s = opaque {};
 pub const uws_header_iterator_s = opaque {};
 pub const uws_app_t = uws_app_s;
-
 pub const uws_socket_context_s = opaque {};
 pub const uws_socket_context_t = uws_socket_context_s;
+
 pub const AnyWebSocket = union(enum) {
     ssl: *NewApp(true).WebSocket,
     tcp: *NewApp(false).WebSocket,
 
-    pub fn raw(this: AnyWebSocket) *RawWebSocket {
+    pub inline fn raw(this: AnyWebSocket) *RawWebSocket {
         return switch (this) {
             .ssl => this.ssl.raw(),
             .tcp => this.tcp.raw(),
         };
     }
-    pub fn as(this: AnyWebSocket, comptime Type: type) ?*Type {
+    pub inline fn as(this: AnyWebSocket, comptime Type: type) ?*Type {
         @setRuntimeSafety(false);
         return switch (this) {
             .ssl => this.ssl.as(Type),
             .tcp => this.tcp.as(Type),
         };
     }
 
-    pub fn close(this: AnyWebSocket) void {
-        const ssl_flag = @intFromBool(this == .ssl);
-        return uws_ws_close(ssl_flag, this.raw());
+    pub inline fn is_ssl(this: AnyWebSocket) u1 {
+        return if (comptime @hasDecl(@TypeOf(this), "ssl")) 1 else 0;
```

Comment:
this should fail to compile

Reply:
oh actually

it just always returns 1

Reply:
so this will segfault for tcp connections most likely

Reply:
I removed all the changes to `uws.zig`, as it wasn't necessary for this PR.